### ISSUE TITLE
Getting WCSLIB dependency from remote server

### ISSUE THREAD

**Issue by SpaceMeerkat** [SoFiA-Admin/SoFiA-2#68]
Not really a SoFiA issue per say; but: is there a way to get the dependency, WCSLIB, from the [source page](https://www.atnf.csiro.au/people/mcalabre/WCS/) when working on a remote server?

I noticed in the page's html that the location of the package files is in the form of an ftp request:
> ftp://ftp.atnf.csiro.au/pub/software/wcslib/wcslib-7.7.tar.bz2

Does this mean there may be a way to grab the files remotely using ftp locally?

Any help on this would be greatly appreciated!

**Comment by vdhulst** [comment]
Not sure I fully understand the question, but from anywhere one can ftp to ftp.atnf.csiro.au and login as anonymous. Then go to pub/software/wcslib/ and get wcslib-7.7.tar.bz2 or earlier versions. The next stap of course is to unpack the tarball and install wcslib.

**Comment by SoFiA-Admin** [comment]
@SpaceMeerkat: I don’t think you need `ftp` to be able to download the file, as any download tool that supports the FTP protocol should be able to retrieve the file. I just tested

```
wget ftp://ftp.atnf.csiro.au/pub/software/wcslib/wcslib-7.7.tar.bz2
```

and this seems to work perfectly fine on my computer.

**Comment by SpaceMeerkat** [comment]
Thanks both for the reply, that works fine on my machines too after all!
